### PR TITLE
added messageSettings component

### PR DESCRIPTION
**Pull request by hrahul2605**
PR to issue #921

### CODE REVIEW

**Comment by Dishebh** [comment]
@hrahul2605 Please add a heading on the top as well saying `Message Settings`, similar to the one you see on `Message Cards` component's page.

**Comment by hrahul2605** [comment]
> @hrahul2605 Please add a heading on the top as well saying `Message Settings`, similar to the one you see on `Message Cards` component's page.

Okay will do that.

**Comment by hrahul2605** [comment]
@Dishebh As you mentioned, I have edited the same. Please take a look.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-MessageSettings/styles.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+export const CheckContainer = styled.div<{ selected?: boolean }>`
+  width: 24px;
+  height: 24px;
+  background: ${props => (props.selected ? C.SecondaryColor : C.GreyLight)};
+  border-radius: 5px;
+  margin-right: 13px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```
Comment (Dishebh) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-MessageSettings/styles.ts:39]: We already have a `DefaultCheckbox` component [here](https://github.com/Ignitus/Ignitus-client/tree/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultCheckbox). Try making use of this component in replace of above :)

File: src/ignitus-UserInterfaceBook/Components/Organisms/interfaceMessageSettings/Components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { MessageSettings, Heading2 } from '../../../../../ignitus-Shared';
+import { Interface } from '../../../../styles';
+
+export const InterfaceMessageSettings: React.FC = () => {
+  const [anyone, setAnyone] = useState(false);
+  const [connections, setConnections] = useState(false);
+  const [directConnections, setDirectConnections] = useState(false);
+
```
Comment (Dishebh) [inline src/ignitus-UserInterfaceBook/Components/Organisms/interfaceMessageSettings/Components/index.tsx:9]: We won't be needing this after you use the already implemented `DefaultCheckbox` component.

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-MessageSettings/styles.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  height: 2.8rem;
+  background: ${C.SecondaryColor};
+  border-radius: 2.38rem;
+  right: 1.6rem;
+  bottom: 1.9rem;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
```
Comment (Dishebh) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-MessageSettings/styles.ts:53]: Use the already implemented `Button` component here as well

**Review by divyanshu-rawat — COMMENTED** [review]
@hrahul2605 can you please resolve the merge conflicts here, as you and @Dishebh both updated the checkbox component according to your needs, please discuss this among yourself and resolve the conflicts.

**Comment by hrahul2605** [comment]
Fixed the merge conflict, please review @Dishebh @divyanshu-rawat

**Review by Dishebh — APPROVED** [review]

**Pull request merged**